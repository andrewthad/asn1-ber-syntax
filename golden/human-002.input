300B0C04 4164616D A1030201 32
