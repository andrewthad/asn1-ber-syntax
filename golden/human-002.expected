Constructed Universal 16
    ( fromListN 2
        [ Utf8String "Adam"
        , Constructed ContextSpecific 1
            ( fromListN 1 [ Integer 50 ] )
        ]
    )
